asset/snippets/test/samples.test.js

```diff
@@ -259,4 +291,42 @@ describe('quickstart sample tests', () => {
     await metadataTable.delete();
     await resultsTable.delete();
   });
+
+  it('should get effective iam policies successfully', async () => {
+    const assetName = `//storage.googleapis.com/${bucketName}`;
+    const stdout = execSync(`node getBatchEffectiveIamPolicies ${assetName}`);
+    assert.include(stdout, assetName);
+  });
+
+  it('should create saved query successfully', async () => {
+    const description = 'description';
+    await createSavedQuery(queryId, description);
+    assert.strictEqual(console.log.callCount, 6);
```

Comment:
Nit: this line is probably overkill for two reasons:
- It's {not very useful, hard to debug} if it breaks in CI.
- In general, we care more about the _args_ to `console.log` than the call-count.

Reply:
removed